diff --git a/src/Makefile b/src/Makefile
index 41a6765..f102ad2 100755
--- a/src/Makefile
+++ b/src/Makefile
@@ -5,7 +5,7 @@ else
 endif
 
 .PHONY: do_all cd cdpdf cdgl cdim cdcontextplus cdlua5 cdluapdf5 cdluagl5 cdluacontextplus5 cdluaim5
-do_all:        cd cdpdf cdgl cdim cdcontextplus cdlua5 cdluapdf5 cdluagl5 cdluacontextplus5 cdluaim5
+do_all:        cd cdgl cdim cdcontextplus 
 
 cd:
 	@$(TECMAKE_CMD)
diff --git a/tecmake.mak b/tecmake.mak
index 48991a4..4e83787 100755
--- a/tecmake.mak
+++ b/tecmake.mak
@@ -1823,7 +1823,7 @@ depend: $(DEPEND)
 $(DEPEND): $(MAKENAME) $(DEPENDDIR)
   ifdef SRC
 	  @echo "" > $(DEPEND)
-	  @which $(CPPC) 2> /dev/null 1>&2 ;\
+	  @true ;\
 	  if [ $$? -eq 0 ]; then \
 	    echo "Tecmake: Building Dependencies ... [ $(DEPEND) ] (can be slow)" ;\
 	    $(CPPC) $(DEPFLAGS) $(DEPINCS) $(STDINCS) $(PKGINCS) $(DEFINES) $(STDDEFS) -MM $(SOURCES) | \
